### ISSUE TITLE
libnetconf2 unable to load modules with fully if-featured enums

### ISSUE THREAD

**Issue by syyyr** [CESNET/libnetconf2#358]
HI, if the NETCONF server has this schema:
```yang
module all_disabled_enums {
  namespace "http://example.com";
  prefix "ds";
  yang-version 1.1;

  feature "myFeature";

  leaf myLeaf {
    type enumeration {
      enum A {
        if-feature "myFeature";
      }
    }
  }
}
```
libnetconf2 is unable to retrieve it, even though the server has the "myFeature" feature enabled. sysrepoctl is able to import this kind of module (although you do have to specify enable the features immediately). This happens even with netopeer2-cli, where this kind of situation does not print any kind of error/warning message even when using the debug verbosity. Then, when using the `get` command, the output can look like this:
```json
  "myLeaf": [
        {}
  ]
```

**Comment by michalvasko** [comment]
Right, the problem was quite straightforward as was the fix but I noticed other issues that were not that simple fix. So, please update *libyang* and *libnetconf2* (optionally *netopeer2*) and this problem should be fixed and loading YANG modules when creating new NETCONF session should be faster, with enough modules the difference may be considerable.

**Comment by syyyr** [comment]
Okay, I tested and it seems to work. Thanks!
